data/hai/hai reveal 5 philosophers.txt

```diff
@@ -212,22 +212,72 @@ mission "Hai Reveal [B03] Contact Giti"
+			label briefing
+			choice
+				`	"What is the status of the <origin> operation?"`
+					goto operation
+					to display
+						not "Hai Reveal [B03] Contact Giti: label operation"
+				`	"Is the Hai disinformation campaign still working?"`
+					goto disinformation
+					to display
+						not "Hai Reveal [B03] Contact Giti: label disinformation"
+				`	"Sayari and Teeneep, what are your plans?"`
+					goto squirrels
+					to display
+						has "Hai Reveal [B03] Contact Giti: label operation"
+						not "Hai Reveal [B03] Contact Giti: label squirrels"
+				`	"I'm honored to have served you as fleet commander."`
+					goto honored
+					to display
+						has "Hai Reveal [B03] Contact Giti: label operation"
+						has "Hai Reveal [B03] Contact Giti: label disinformation"
+						has "Hai Reveal [B03] Contact Giti: label squirrels"
+				`	"Excellent work, everyone! Carry on!"`
+					goto offended
+					to display
+						has "Hai Reveal [B03] Contact Giti: label operation"
+						has "Hai Reveal [B03] Contact Giti: label disinformation"
+						has "Hai Reveal [B03] Contact Giti: label squirrels"
+
+			label operation
+			action
+				set "Hai Reveal [B03] Contact Giti: label operation"
+			`	Danforth begins the report. "Stage one has been successful. However, it looks like we're going to be here a while. <origin> is a rabbit warren of hidden places. Even with the Hai resources on top of everything we've brought, it could take weeks to root things out sufficiently."`
 			`	"General Choot'k, would you agree with the human's assessment of the effort required here?" asks Teeneep.`
 			`	The Unfettered warlord does not even hesitate to respond, "Yes. Even with our superior capacities we will be here a while."`
+				goto briefing
+			label disinformation
+			action
+				set "Hai Reveal [B03] Contact Giti: label disinformation"
+			`	Batu clears his throat and Danforth glances at him before evidently remembering something discussed earlier. "I'm sorry Captain <last> and Elder Teeneep. I must advise you that at this stage all hope of keeping the Hai a secret is lost. Even if we could convince all factions to keep the secret, the blockade has drawn enough attention so all that remains is to resolve issues before the Hai become irretrievably common knowledge."`
+			`	Teeneep responds, "We all knew that day would come soon. Be glad we're here together to usher it in peacefully and with strength."`
+				goto briefing
+			label squirrels
+			action
+				set "Hai Reveal [B03] Contact Giti: label squirrels"
+			`	"Our place is here, Captain <last>," says Teeneep, definitively.`
 			`	Danforth looks between them. "Perhaps, Elder, you and the Ambassador would find it more comfortable to return to Mountaintop? There will be little diplomacy here, and scarcely more safety, and it would seem to me that even with Batu here as the deputy minister's eyes on the ground, it would be easier to conduct your responsibilities directly."`
```

Comment:
```suggestion
			`	Danforth looks between them. "Perhaps, Elder, you and the Ambassador would find it more comfortable to return to Mountaintop? There will be little diplomacy here, and scarcely more safety, and it would seem to me that, even with Batu here as the Deputy Minister's eyes on the ground, it would be easier to conduct your responsibilities directly."`
```